v2/thirdpartypasswordless/common-customizations/getting-provider-access-token.mdx

```diff
@@ -0,0 +1,184 @@
+---
+id: getting-provider-access-token
+title: Getting provider's access token
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./thirdpartyemailpassword/post-login/getting-provider-access-token.mdx -->
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import TabItem from '@theme/TabItem';
+
+# Getting provider's access token
+
+You can get the Third Party Provider's access token to query their APIs with the following method:
+
+<BackendSDKTabs>
+<TabItem value="nodejs">
+
+```tsx
+import SuperTokens from "supertokens-node";
+import ^{recipeNameCapitalLetters} from "supertokens-node/recipe/^{codeImportRecipeName}";
+import Session from "supertokens-node/recipe/session";
+
+SuperTokens.init({
+    appInfo: {
+        apiDomain: "...",
+        appName: "...",
+        websiteDomain: "..."
+    },
+    supertokens: {
+        connectionURI: "...",
+    },
+    recipeList: [
+        ^{recipeNameCapitalLetters}.init({
+            ^{nodeRecipeInitDefault}
```

Comment:
Here you don't need to use `nodeRecipeInitDefault` and can just add a `@ts-ignore` annotation